### ISSUE TITLE
False Positive for Credo.Check.Readability.StrictModuleLayout when using Surface prop called import

### ISSUE THREAD

**Issue by maennchen** [rrrene/credo#872]
### Precheck

* [x] Proposals for new features should be submitted via: https://github.com/rrrene/credo-proposals
* [x] For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): `1.5.5-ref.update_deps.ac0908d`
* Erlang/Elixir version (`elixir -v`):
```
Erlang/OTP 23 [erts-11.2] [source] [64-bit] [smp:6:6] [ds:6:6:10] [async-threads:1] [hipe]

Elixir 1.11.4 (compiled with Erlang/OTP 21)
```
* Operating system: `Ubuntu 20.04.2 LTS`

### What were you trying to do?

Checking the following file using credo:

```elixir
defmodule HygeiaWeb.ImportLive.Header do
  @moduledoc false

  use HygeiaWeb, :surface_live_component

  alias Hygeia.ImportContext.Import
  alias Hygeia.ImportContext.Import.Type
  alias HygeiaWeb.UriActiveContext
  alias Surface.Components.LiveRedirect

  prop import, :map, required: true
end
```

This file is using [`surface`](https://github.com/surface-ui/surface) and is defining a prop called `import`.

### Expected outcome

No error

### Actual outcome

```
  Code Readability                                                                                                                                                                                                                            
┃ 
┃ [R] ↘ import must appear before alias
┃       apps/hygeia_web/lib/hygeia_web/live/import_live/header.ex:11:8 #(HygeiaWeb.ImportLive.Header)
```

**Comment by wingyplus** [comment]
I try to reproduce with small test case:

```diff
diff --git a/test/credo/check/readability/strict_module_layout_test.exs b/test/credo/check/readability/strict_module_layout_test.exs
index 2c49492d..437c193e 100644
--- a/test/credo/check/readability/strict_module_layout_test.exs
+++ b/test/credo/check/readability/strict_module_layout_test.exs
@@ -257,4 +257,30 @@ defmodule Credo.Check.Readability.StrictModuleLayoutTest do
       assert issue.message == "alias must appear before require"
     end
   end
+
+  test "no errors on import that isn't import statement" do
+    """
+    defmodule Test1 do
+      alias Foo
+
+      def import(), do: nil
+    end
+    """
+    |> to_source_file
+    |> run_check(@described_check)
+    |> refute_issues
+
+    """
+    defmodule Test2 do
+      use Baz
+
+      alias Foo
+
+      prop import, :map, required: true
+    end
+    """
+    |> to_source_file
+    |> run_check(@described_check)
+    |> refute_issues
+  end
 end
```

The first source code, `Credo.Code.Module.analyze()` produces:

```elixir
[{Test1, [alias: [line: 2, column: 3], public_fun: [line: 4, column: 3]]}]
```

And the second code:

```elixir
[
  {Test2,
   [
     use: [line: 2, column: 3],
     alias: [line: 4, column: 3],
     import: [line: 6, column: 8]
   ]}
]
```

**Comment by wingyplus** [comment]
Another example (copied from Nx) is fail also:

```elixir
defmodule Test3 do
  use Baz

  alias Foo

  defn import(t), do: t
end

**Comment by wingyplus** [comment]
The `Macro.prewalk` traversing the node when it found: 

```elixir
{:prop, [line: 6, column: 3],
 [{:import, [line: 6, column: 8], nil}, :map, [required: true]]}
```

Then the analyze in `Credo.Code.Module.analyze/2` found the import statement. That causes false trigger.

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by rrrene** [comment]
This is part of v1.6.1 💚
